src/api/concat.cc

```diff
@@ -37,23 +37,23 @@ void TaintConcatOperator(const FunctionCallbackInfo<Value>& args) {
         return;
     }
 
-    auto transaction = GetTransaction(utils::GetLocalStringPointer(args[0]));
+    auto transaction = GetTransaction(utils::GetLocalPointer(args[0]));
```

Comment:
most of the changes come from the refactor of the `GetLocalStringPointer` method to `GetLocalPointer`

Reply:
The refactor makes sense